src/test/java/org/prebid/server/bidder/vox/VoxBidderTest.java

```diff
@@ -0,0 +1,203 @@
+        final var httpCall = givenHttpCall(
+                givenBidRequest(identity()),
+                givenBidResponse(audioBid)
+        );
+
+        // when
+        final var result = sut.makeBids(httpCall, null);
+        // then
+        assertThat(result.getErrors())
+                .containsExactly(badServerResponse("Unable to fetch mediaType in multi-format: id"));
+    }
+
+    private static Result<List<HttpRequest<BidRequest>>> expectedResults(BidRequest bidRequest) {
+        return Result.withValue(HttpRequest.<BidRequest>builder()
+                .method(HttpMethod.POST)
+                .uri(ENDPOINT_URL)
+                .headers(expectedHeaders())
+                .body(jacksonMapper.encodeToBytes(bidRequest))
+                .payload(bidRequest)
+                .build()
+        );
+    }
+
+    private static MultiMap expectedHeaders() {
+        return headers()
+                .set(CONTENT_TYPE_HEADER, APPLICATION_JSON_CONTENT_TYPE)
+                .set(ACCEPT_HEADER, APPLICATION_JSON_VALUE);
+    }
+
+    private static BidRequest givenBidRequest(Function<Imp.ImpBuilder, Imp.ImpBuilder>... impCustomizers) {
+        return givenBidRequest(identity(), List.of(impCustomizers));
+    }
+
+    private static BidRequest givenBidRequest(
+            Function<BidRequest.BidRequestBuilder, BidRequest.BidRequestBuilder> bidRequestCustomizer,
+            List<Function<Imp.ImpBuilder, Imp.ImpBuilder>> impCustomizers) {
+        return bidRequestCustomizer.apply(
+                BidRequest.builder().imp(impCustomizers.stream().map(VoxBidderTest::givenImp).toList())
+        ).build();
+    }
+
+    private static Imp givenImp(Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer) {
+        return impCustomizer.apply(Imp.builder()
+                        .id("imp_id")
+                        .ext(mapper.valueToTree(ExtPrebid.of(
+                                null,
+                                new VoxImpExt("placementId", "http://some.image.url", List.of("1920x1080")))
+                        )))
+                .build();
+    }
+
+    @SneakyThrows
+    private static String givenEmptyBidResponse() {
+        return mapper.writeValueAsString(BidResponse.builder().build());
+    }
+
+    @SneakyThrows
+    private static String givenBidResponse(Bid... bids) {
+        return mapper.writeValueAsString(
+                BidResponse.builder()
+                        .cur(CURRENCY)
+                        .seatbid(bids.length == 0 ? List.of() : List.of(SeatBid.builder().bid(List.of(bids)).build()))
+                        .build()
+        );
+    }
+
+    private static BidderCall<BidRequest> givenHttpCall(BidRequest bidRequest, String body) {
+        return BidderCall.succeededHttp(HttpRequest.<BidRequest>builder().payload(bidRequest).build(),
+                HttpResponse.of(200, null, body),
+                null
+        );
+    }
+
+    @Value
+    private static class VoxImpExt {
```

Comment:
This should be under org.prebid.server.proto.openrtb.ext.request.vox

Reply:
it's a test class and one-time usage of this object in the test, are sure do we need to move to a separate class/package

Reply:
so this class won't be used in the prod code, only in tests, generally it's not a good approach, let's add it only when it's necessary

Reply:
In cases like this, we have three approaches 
1) Your approach
2) Just create mapper.createObjectNode()(as we don't have any validations for props inside bidder)
3) Create class under package defined for bidder extensions(This way is working go team)

Now, it's your choice)